.github/workflows/pypi-publish.yaml

```
@@ -0,0 +1,66 @@
+        run: chmod +x ./scripts/install_dependencies.sh && ./scripts/install_dependencies.sh
+
+      - name: Set version from git tag
+        run: |
+          VERSION="${GITHUB_REF_NAME#v}"
+          sed -i "s/^version = \"0.0.0\"/version = \"${VERSION}\"/" pyproject.toml
+
+      - name: Build pycer wheel
+        run: uv build --wheel -v 2>&1 | tee /tmp/pycer-build.log
+
+      - name: Verify logging is set to info
+        run: grep -q "Logging level: info" /tmp/pycer-build.log || { echo "::error::LOGGING was not set to info during CMake configure"; exit 1; }
```

actionlint (1.7.11)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)